functions-app/report-psychometrician/service/anomaly-report.service.js

```diff
@@ -351,9 +355,15 @@ anomalyReportService.detectQuestionsThatWereShownForTooLong = (check) => {
   const audits = anomalyReportService.filterAllRealQuestionsAndPauseAudits(check)
   const config = check.data.config
   const head = R.head(audits)
+  if (!head) {
+    return
+  }
   const tail = R.tail(audits)
+  if (!tail) {
```

Comment:
@jon-shipley that can also mean that you return if you have a single item within the array of audits.
Please confirm if that is expected behavior

Reply:
Removed - thanks!